apps/kimai/config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Kimai",
+  "available": true,
+  "port": 8003,
+  "exposable": true,
+  "id": "kimai",
+  "description": "Kimai is a professional grade time-tracking application, free and open-source. It handles use-cases of freelancers as well as companies with dozens or hundreds of users.",
+  "tipi_version": 1,
+  "version": "2.0.35",
```

Comment:
Here you should use the kimai container version. So it would be this: ``fpm-2.0.35-prod``